tests/run.py

```diff
@@ -0,0 +1,1604 @@
+"""
```

Comment:
This is a python version of the test that has a number of problems:
1. it doesn't actually compare, just prints
2. has bad comments
3. numbers are way too long
4. it doesn't run as part of the suite
5. it's named badly
6. isn't a function

If you want, you could fix these issues to make a python version of the same test. For example, you could change the number array to a string and convert that to numbers to send to ypy. But I think you should just delete this file. Or write your own python e2e test. But probably just delete.

One good test for python would be two convert some JSON all the full cycle (from nodes to an update and back to JSON). That would probably be a better test and nicer than this file. But you'll need the reverse function first obviously.

Reply:
Moved it to `test_lexical_parse.py`